README.md

```diff
@@ -1,24 +1,22 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+### Setup Instructions
+This following steps are instructions to launch and view a simple Ruby on Rails applicaiton that authenticates with Procore's API using the OAuth 2.0 Authorization Code Grant Type flow. The application is configured to access Procore's environment. If you do not have access to this environment or if you're working out of Procore's developer sandbox environment, all root URLs in the `login_controller` and `users_controller` must be re-configured to https://sandbox.procore.com.
```

Comment:
We should make it so the developer doesn't need to go into the controllers to do this. Better to have the environment's base URL be in the application.yml file that they're already going to need to modify for this example app.

Reply:
Reconfigured controllers for environment flexibility and updated the README with new instructions on setup.